enhancements/20220525-system-logging.md

```diff
@@ -0,0 +1,65 @@
+# System Logging
+
+## Summary
+
+We need to be able to support exporting harvester system logs outside the cluster, to allow for collecting and analyzing those logs to aid in debugging and harvester management. 
```

Comment:
Please have a definiton/scope of the to-be collected and listed logs.
e.g. 
each POD/container logs
various of rancher/... systemctl serices logs
kubelet/api server/kube-proxy logs

What's the definition of log kind/type/level.